### DISCUSSION TITLE
Remove the Html extension from the URL

### DISCUSSION THREAD

**Discussion by Fernando-96** [vercel/community#679] [Help] [answered]
I want to remove the Html extension from the URL on my website. The website is deployed through Vercel meaning I can't use the .htacces file to do it.

Can some1 tell me how to solve this situation?

**Comment by Fernando-96** [comment, marked as answer]
I Got the answer under [cleanUrls](https://vercel.com/docs/project-configuration#project-configuration/clean-urls)
create a vercel.json file in your root directory then add the following
 {
  "cleanUrls": true
}

**Reply by amyegan** [reply]
I'm glad you were able to find the answer. Thanks for sharing it here!
